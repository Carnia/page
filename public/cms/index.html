<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>自定义项目名</title><link rel="icon" href="./favicon.ico"><script defer="defer" src="./js/main.fb2360d0.js"></script><link href="./css/main.css" rel="stylesheet"></head><body><div id="root"></div></body></html>