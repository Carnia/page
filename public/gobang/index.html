<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta name=format-detection content="telephone=no, email=no"><meta name=viewport content="user-scalable=no,width=device-width,initial-scale=1,maximum-scale=1,viewport-fit=cover"><meta http-equiv=X-UA-Compatible content="ie=edge"><link rel=icon href=/public/gobang/favicon.ico><title>gobang</title><link href=/public/gobang/js/app.68000ea8.js rel=preload as=script><link href=/public/gobang/js/chunk-vendors.436d6070.js rel=preload as=script></head><body><noscript><strong>We're sorry but gobang doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/public/gobang/js/chunk-vendors.436d6070.js></script><script src=/public/gobang/js/app.68000ea8.js></script></body></html>