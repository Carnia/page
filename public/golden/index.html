<!DOCTYPE html><html lang=en><head><meta charset=UTF-8><meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no"><meta http-equiv=X-UA-Compatible content="ie=edge"><title></title><link rel=icon href=/favicon.ico type=image/x-icon><style>* {
      margin: 0;
      padding: 0;
    }</style><script>var _html=document.documentElement;function getSize(){var e=_html.clientWidth;_html.style.fontSize=e<=320?"17px":750<=e?"40px":e/750*40+"px"}getSize();var timer=null;window.addEventListener("resize",function(e){clearTimeout(timer),timer=setTimeout(function(){getSize()},300)})</script></head><body><div id=app></div><script>!function(f){function e(e){for(var r,t,n=e[0],o=e[1],u=e[2],l=0,i=[];l<n.length;l++)t=n[l],Object.prototype.hasOwnProperty.call(a,t)&&a[t]&&i.push(a[t][0]),a[t]=0;for(r in o)Object.prototype.hasOwnProperty.call(o,r)&&(f[r]=o[r]);for(s&&s(e);i.length;)i.shift()();return c.push.apply(c,u||[]),p()}function p(){for(var e,r=0;r<c.length;r++){for(var t=c[r],n=!0,o=1;o<t.length;o++){var u=t[o];0!==a[u]&&(n=!1)}n&&(c.splice(r--,1),e=l(l.s=t[0]))}return e}var t={},a={1:0},c=[];function l(e){if(t[e])return t[e].exports;var r=t[e]={i:e,l:!1,exports:{}};return f[e].call(r.exports,r,r.exports,l),r.l=!0,r.exports}l.m=f,l.c=t,l.d=function(e,r,t){l.o(e,r)||Object.defineProperty(e,r,{enumerable:!0,get:t})},l.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},l.t=function(r,e){if(1&e&&(r=l(r)),8&e)return r;if(4&e&&"object"==typeof r&&r&&r.__esModule)return r;var t=Object.create(null);if(l.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:r}),2&e&&"string"!=typeof r)for(var n in r)l.d(t,n,function(e){return r[e]}.bind(null,n));return t},l.n=function(e){var r=e&&e.__esModule?function(){return e.default}:function(){return e};return l.d(r,"a",r),r},l.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},l.p="/public/golden";var r=window.webpackJsonp=window.webpackJsonp||[],n=r.push.bind(r);r.push=e,r=r.slice();for(var o=0;o<r.length;o++)e(r[o]);var s=n;p()}([])</script><script type=text/javascript src=/public/golden/statics/js/vendors_app.7828a090.js></script><script>(window.webpackJsonp=window.webpackJsonp||[]).push([[0],{bzer:function(e,i,n){"use strict";n.r(i);var o=Boolean("localhost"===window.location.hostname||"[::1]"===window.location.hostname||window.location.hostname.match(/^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/));function a(e){navigator.serviceWorker.addEventListener("controllerchange",function(){window.location.reload()}),navigator.serviceWorker.register(e).then(function(i){i.onupdatefound=function(){var e=i.installing;e.onstatechange=function(){"installed"===e.state&&navigator.serviceWorker.controller}}}).catch(function(e){})}var k=n("STO9"),t=(k.h.TextureCache,n("cvzg")),d=n.n(t),r=n("qtOx"),l=n.n(r),s=function(e,i){},c=n("9553"),h=n.n(c),g="/".concat("public/golden","/statics/images/golden/"),u={bg1:{url:g+"background-1.jpg"},bg2:{url:g+"background-2.png"},logo:{url:g+"logo.png"},boom:{url:g+"boom.png"},close:{url:g+"close.png"},gold1:{url:g+"gold_1.png"},gold2:{url:g+"gold_2.png"},gold3:{url:g+"gold_3.png"},gold4:{url:g+"gold_4.png"},car:{url:g+"golden_car.png"},hook:{url:g+"golden_hook.png"},luckyBag:{url:g+"lucky_bag.png"},coinsContainer:{url:g+"coins_container.png"},halo:{url:g+"halo.png"},dialogBoom:{url:g+"dialog_boom.png"},dialogBag:{url:g+"dialog_bag_golden_full.png"},goldenFloat:{url:g+"golden_float.png"}},p={loading_house:{url:g+"loading_house.png"},loading_golden:{url:g+"loading_golden.png"},loading_car:{url:g+"loading_car.png"},loading_bg:{url:g+"loading_bg.jpg"}},w=new k.b,f=new k.c;f.beginFill(16750899),f.drawRoundedRect(0,0,.6*window.innerWidth,20,12),f.endFill();var v=new k.c;v.lineStyle(4,16750899,1),v.beginFill(16777215),v.drawRoundedRect(0,0,.6*window.innerWidth,20,12),v.endFill(),w.addChild(v),w.addChild(f),w.innerBar=f;var m=w,x=n("QVg2"),y=n("Vo3s"),b="WebGL";k.h.isWebGLSupported()||(b="canvas"),k.h.sayHello(b),new function e(){var b=this;d()(this,e),l()(this,"app",void 0),l()(this,"bump",new h.a),l()(this,"_designWidth",750),l()(this,"_designHeight",1624),l()(this,"w",void 0),l()(this,"h",void 0),l()(this,"state",void 0),l()(this,"scale",void 0),l()(this,"loadingProcess",0),l()(this,"maxLoadingL",0),l()(this,"resources",void 0),l()(this,"LoadingWrapper",void 0),l()(this,"SenceWrapper1",void 0),l()(this,"pause",!1),l()(this,"digging",!1),l()(this,"growing",!0),l()(this,"rotateSpeed",.005),l()(this,"rotateDirection",1),l()(this,"hookSpeed",2),l()(this,"hookDirection",1),l()(this,"initLen",void 0),l()(this,"score",0),l()(this,"sprites",{}),l()(this,"catched",void 0),l()(this,"eventLock",!1),l()(this,"goldenArea",{row:3,column:4,initWidth:720,initHeight:680}),l()(this,"makeSenceLoading",function(e,i){var n=b.w,o=b.h,a=b.LoadingWrapper=new k.b,t=new k.e(i[p.loading_bg.url].texture),d=new k.e(i[p.loading_car.url].texture),r=new k.e(i[p.loading_golden.url].texture),l=new k.e(i[p.loading_house.url].texture),s=new k.g({fontFamily:"Arial",fontSize:16,fill:"orange"}),c=new k.f("loading...",s),h=b.scale;t.width=n,t.height=o,Object.assign(r,{x:.5*n,y:.25*o,scale:{x:h,y:h},anchor:{x:.5,y:.5}}),Object.assign(m,{x:.2*n,y:.6*o,anchor:{x:.5,y:.5}}),Object.assign(d,{x:.2*n,y:.6*o,scale:{x:h,y:h},anchor:{x:.5,y:.5}}),Object.assign(c,{x:.5*n,y:.7*o,anchor:{x:.5,y:.5}}),l.anchor.set(.5,.5);var g=.8*n/l.width;l.scale.set(g,g),l.position.set(.5*n,.46*o),a.addChild(t),a.addChild(l),a.addChild(c),a.addChild(m),a.addChild(d),a.addChild(r),b.app.stage.addChild(a),b.loadImg(),b.state=function(){b.maxLoadingL=(.2+b.loadingProcess/100*.6)*n,c.text=b.loadingProcess+"%",m.innerBar.width=b.loadingProcess/100*.6*n,10<b.maxLoadingL-d.x?d.x+=10:d.x=b.maxLoadingL,100===b.loadingProcess&&d.x===b.maxLoadingL&&(m.innerBar.width=.6*n,b.state=function(){})},b.app.ticker.add(function(e){b.state()})}),l()(this,"loadImg",function(){new Promise(function(n,i){var o=new k.d;Object.keys(u).forEach(function(e){o.add(e,u[e].url)}),o.load(function(e,i){b.resources=i,n(i)}),o.onError.add(function(e){i(e)}),o.onLoad.add(function(e){b.loadingProcess=e.progress})}).then(b.makeSence1)}),l()(this,"makeSence1",function(){var e=b.scale,i=b.app,n=b.sprites,o=b.resources;b.LoadingWrapper.visible=!1;var a=b.SenceWrapper1=new k.b;Object.keys(o).forEach(function(e){b.sprites[e]=new k.e(o[e].texture)});var c=b.makeBackground(),h=b.makeDialog(),g=b.makeRopeWrapper(),t=n.car,u=n.hook,d=new k.b;Object.assign(d,{x:344*e,y:100*e}),t.scale.set(e),d.addChild(t),d.addChild(g);var p=b.makeWallsWrapper(),w=b.makeItemsWrapper();a.addChild(c),a.addChild(d),a.addChild(p),a.addChild(w),a.addChild(h),i.stage.addChild(a),a.interactive=!0,a.on("pointerdown",function(){b.eventLock||b.digging||(b.digging=!0,b.growing=!0,b.hookSpeed=2,b.hookDirection=1,g.rope.height+=1)}),b.state=function(){var e=b.pause,i=b.digging,n=(b.growing,b.rotateSpeed),o=b.rotateDirection,a=b.hookSpeed,t=b.hookDirection,d=b.bump,r=b.catched,l=b.initLen,s=g.rope;e||(i?(s.height+=a*t,u.y+=a*t,r&&(r.x=u.x,r.y=u.y+u.height/2),s.height<=l&&(r&&(0<r.score?h.showMsg(r.endPic,"恭喜你!","获得了".concat(r.score,"个金币")):h.showMsg(r.endPic,"哎呀 碰到地雷了"),b.score+=r.score,c.renderScore(b.score.toString()),r.tl.kill(),b.catched=r.destroy()),b.digging=!1)):(g.rotation+=n*o,(.8<=g.rotation||g.rotation<=-.8)&&(o=b.rotateDirection*=-1)),p.children.forEach(function(e){d.hit(u,e,!1,!1,!0)&&(b.growing=!1,b.hookDirection=-1)}),i&&w.children.forEach(function(e){d.hit(u,e,!1,!1,!0)&&(b.growing=!1,b.hookSpeed*=.5,b.hookDirection=-1,g.addChild(e),b.catched=e)}))},i.ticker.add(function(){b.state()})}),l()(this,"makeDialog",function(){var i=b.scale,n=b.w,o=b.h,a=b.sprites,e=a.gold1,t=a.gold2,d=a.gold3,r=a.gold4,l=a.dialogBoom,s=a.dialogBag,c=a.goldenFloat,h=a.halo,g=a.close,u=[e,t,d,r,l,s];e._scale=1,t._scale=1,d._scale=.8,r._scale=.7;var p=new k.b;p.visible=!1,p.hide=function(){clearTimeout(p.timer),b.eventLock=!1,p.visible=!1},[h].concat(u).forEach(function(e){p.addChild(e),e.anchor.set(.5),e.scale.set(e._scale||i),e.position.set(n/2,o/2-100*i)});var w=new k.g({fontFamily:"PingFangSC-Regular, sans-serif",fontWeight:"bold",fontSize:62,lineHeight:62,fill:"#ffffff",stroke:"#c97722",strokeThickness:8}),f=new k.g({fontFamily:"PingFangSC-Regular, sans-serif",fontWeight:"bold",fontSize:43,lineHeight:43,fill:"#ffebac",stroke:"#c97722",strokeThickness:10});p.addChild(g),g.scale.set(i),g.anchor.set(.5),g.interactive=!0,g.on("pointertap",p.hide);var v=new k.f("恭喜你!",w);v.scale.set(i),v.anchor.set(.5),v.position.set(n/2,o/2+75);var m=new k.f("获得了1个金币",f);return m.scale.set(i),m.anchor.set(.5),m.position.set(n/2,o/2+110),p.addChild(v),p.addChild(m),p.text1=v,p.text2=m,g.position.set(p.width/2+h.width/2,p.height/2-100*i),p.showMsg=function(e,i,n){u.forEach(function(e){return e.visible=!1}),a[e].visible=!0,p.visible=!0,v.text=i,m.text=n||"",b.eventLock=!0,n&&function(e,i){var n=2<arguments.length&&void 0!==arguments[2]?arguments[2]:.5;i.scale.set(n,n),i.position.set(120*n,120*n),i.alpha=0,e.addChild(i);var o=new x.c;o.add(y.a.to(i,1,{alpha:1,y:90*n})),o.add(y.a.to(i,.5,{alpha:0,y:60*n})),o.play()}(p,c),p.timer=setTimeout(function(){p.hide()},1500)},p}),l()(this,"makeBackground",function(){var e=b.scale,i=b.sprites,n=b._designHeight,o=b.w,a=b.h,t=i.bg1,d=i.bg2,r=i.coinsContainer,l=i.logo,s=new k.b,c=new k.b,h=new k.f("0",new k.g({fontSize:18,fill:"gold"}));t.width=o,t.height=a,d.width=o;var g=a*(n-1297)/n;return d.height=1297*e,d.y=g,l.scale.set(e),l.position.set(120*e,130*e),c.addChild(r),c.addChild(h),r.scale.set(e),c.x=20*e,c.y=30*e,h.x=40,h.y=.1*r.height,s.addChild(t),s.addChild(d),s.addChild(l),s.addChild(c),s.renderScore=function(e){h.text=e},s}),l()(this,"makeRopeWrapper",function(){var e=b.scale,i=b.sprites.hook,n=new k.b,o=new k.c,a=50*e;o.beginFill(6567711),o.drawRect(0,0,4*e,a),o.endFill(),Object.assign(n,{x:28*e,y:150*e}),n.addChild(o),n.addChild(i);var t=n.width;return o.x=t/2-o.width/2,i.anchor.set(.5,.5),i.scale.set(.6*e),Object.assign(i,{x:t/2,y:a+i.height/2}),n.pivot.x=t/2,n.pivot.y=0,n.rope=o,n}),l()(this,"makeWallsWrapper",function(){var e=new k.b,i=b.w,n=b.h,o=new k.e;o.width=1,o.height=n,o.x=-1,o.y=0;var a=new k.e;a.width=1,a.height=n,a.x=i,a.y=0;var t=new k.e;return t.width=i,t.height=1,t.x=0,t.y=n,e.addChild(o),e.addChild(a),e.addChild(t),e}),l()(this,"makeItemsWrapper",function(){var e=b.scale,i=b.resources,n=b._designHeight,o=b._designWidth,a=(b.w,b.h*(n-1297)/n),t=new k.b;t.x=0,t.y=a;var d,r,l,s,c,h,g=(o-720)/2,u=450*e,p=[{name:"gold1",score:1,endPic:"gold1",endPicScale:2},{name:"gold2",score:2,endPic:"gold2",endPicScale:2},{name:"gold3",score:3,endPic:"gold3",endPicScale:2},{name:"gold4",score:4,endPic:"gold4",endPicScale:2},{name:"luckyBag",score:6,endPic:"dialogBag",endPicScale:2},{name:"boom",score:-1,endPic:"dialogBoom",endPicScale:2}];for(var w=0;w<16;w++){var f=parseInt(w/4+"",10),v=(g+w%b.goldenArea.column*180+90*Math.random())*b.scale,m=(u+195*f+97.5*Math.random())*b.scale;t.addChild((d=v,r=m,h=s=l=void 0,c=p[Math.floor(Math.random()*p.length)],h=new k.e(i[c.name].texture),Object.assign(h,c),h.anchor.set(e),h.scale.set(.3+.2*Math.random()),h.rotation=2*Math.PI*Math.random(),h.x=d+h.width/2,h.y=r+h.height/2,l=h,(s=new x.c).add(y.a.to(l,1,{width:1.1*l.width,height:1.1*l.height,repeat:-1,yoyo:!0,delay:2*Math.random()})),s.play(),l.tl=s,h))}return t});var i=this.w=window.innerWidth,n=this.h=window.innerHeight;this.app=new k.a({width:i,height:n,forceCanvas:!0,resolution:2,antialias:!0,autoResize:!0}),this.scale=i/this._designWidth,this.initLen=50*this.scale,document.body.appendChild(this.app.view);var o=this.app.loader.add([p.loading_bg.url,p.loading_car.url,p.loading_golden.url,p.loading_house.url]);o.onProgress.add(s),o.load(this.makeSenceLoading)},"serviceWorker"in navigator&&window.addEventListener("load",function(){var i,e="/serviceWorker.js";o?(i=e,fetch(i).then(function(e){404===e.status||-1===e.headers.get("content-type").indexOf("javascript")?navigator.serviceWorker.ready.then(function(e){e.unregister().then(function(){window.location.reload()})}):a(i)}).catch(function(){}),navigator.serviceWorker.ready.then(function(){})):a(e)})}},[["bzer",1,2]]])</script></body></html>